packages/wallet/test/mock-aggregator.spec.ts

```diff
@@ -44,13 +49,17 @@ describe('MockAggregator', async () => {
 
   describe('applyTransaction', async () => {
     it('should update bobs balance using applyTransaction to send 5 tokens', async () => {
+      const transaction = {
+        tokenType: UNI_TOKEN_TYPE,
+        recipient: 'bob',
```

Comment:
Ah an interesting note is we currently allow people to send to "invalid" addresses like this. I made an issue about this so we don't forget to add a check in the state machine which can filter out these sorts of addresses https://github.com/plasma-group/pigi/issues/436